web/src/components/gnd-feature-type-editor/gnd-sortable.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC
```

Comment:
Copyright 2019

Thanks for refactoring this

Reply:
Done, need LGTM again =)